src/z3c/sqlalchemy/util.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
The part about `__future__` imports being below the topmost block comment applies to this one too, but actually this import is not necessary in this file.  The only use of print() is at the very bottom, guarded by `if __name__ == '__main__':`, and it's a simple `print(expr)` that works in Python 2 just as well as in Python 3.

So maybe just drop the `__future__` import.